libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.html

```diff
@@ -28,6 +28,50 @@
     label="{{
       'dmp.steps.project.input.description' | translate
     }}"></app-textarea-wrapper>
+
+  <div formGroupName="funding">
+    <div class="form-row" formGroupName="funderId">
+      <app-input-wrapper-options
+        class="input-60"
+        [control]="funder"
+        [options]="funderOptions"
+        label="{{ 'Funder' | translate }}"></app-input-wrapper-options>
```

Comment:
There is no translation key "Funder". It should say "Funder ID", to make it more obvious for users what to put in this field.